### PR TITLE
Amended flux for VH

### PR DESCRIPTION
**Pull request by cts-code**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
VIH-7466

### Change description ###
Flux config for VH lower environments.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/environments/demo/common-overlay/video-hearings/vh-aad-identity.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ metadata:
  namespace: vh
 spec:
   type: 0
-  resourceID: /subscriptions/867a878b-cb68-4de5-9741-361ac9e178b6/resourceGroups/managed-identities-dev-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-dev-kvuser
-  clientID: f6449721-dfeb-4656-8f60-3d378bb7b687
+  resourceID: /subscriptions/867a878b-cb68-4de5-9741-361ac9e178b6/resourceGroups/managed-identities-demo-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-demo-kvuser
```
Comment (stravze) [inline k8s/environments/demo/common-overlay/video-hearings/vh-aad-identity.yaml:8]: subscription needs replacing

File: k8s/environments/ithc/common-overlay/video-hearings/vh-aad-identity.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: vh-aad-identity
+ namespace: vh
+spec:
+  type: 0
+  resourceID: /subscriptions/ba71a911-e0d6-4776-a1a6-079af1df7139/resourceGroups/managed-identities-ithc-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-ithc-kvuser
```
Comment (stravze) [inline k8s/environments/ithc/common-overlay/video-hearings/vh-aad-identity.yaml:8]: can you confirm theses
Subscription and ClientID

File: k8s/environments/stg/common-overlay/video-hearings/vh-aad-identity.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: vh-aad-identity
+ namespace: vh
+spec:
+  type: 0
+  resourceID: /subscriptions/74dacd4f-a248-45bb-a2f0-af700dc4cf68/resourceGroups/managed-identities-stg-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-stg-kvuser
```
Comment (stravze) [inline k8s/environments/stg/common-overlay/video-hearings/vh-aad-identity.yaml:8]: same again

File: k8s/environments/test/common-overlay/video-hearings/vh-aad-identity.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+ name: vh-aad-identity
+ namespace: vh
+spec:
+  type: 0
+  resourceID: /subscriptions/3eec5bde-7feb-4566-bfb6-805df6e10b90/resourceGroups/managed-identities-test-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-infra-core-test-kvuser
```
Comment (stravze) [inline k8s/environments/test/common-overlay/video-hearings/vh-aad-identity.yaml:8]: and here

**Review by stravze — CHANGES_REQUESTED** [review]

File: k8s/environments/test/common-overlay/video-hearings/vh-aad-identity.yaml (hunk 4 above)
Comment (cts-code) [reply k8s/environments/test/common-overlay/video-hearings/vh-aad-identity.yaml:8]: These are correct.

File: k8s/environments/stg/common-overlay/video-hearings/vh-aad-identity.yaml (hunk 3 above)
Comment (cts-code) [reply k8s/environments/stg/common-overlay/video-hearings/vh-aad-identity.yaml:8]: These are correct.

File: k8s/environments/ithc/common-overlay/video-hearings/vh-aad-identity.yaml (hunk 2 above)
Comment (cts-code) [reply k8s/environments/ithc/common-overlay/video-hearings/vh-aad-identity.yaml:8]: These are correct.

File: k8s/environments/demo/common-overlay/video-hearings/vh-aad-identity.yaml (hunk 1 above)
Comment (cts-code) [reply k8s/environments/demo/common-overlay/video-hearings/vh-aad-identity.yaml]: Resolved.

**Review by cts-code — COMMENTED** [review]
Amended one subscription ID.

**Review by stravze — APPROVED** [review]

**Pull request merged**